cradle-admin-tool-http/scripts/update-pages-comment.sh

```diff
@@ -0,0 +1,360 @@
+  echo " Arg: ${ARG_MODE} (optional) - work mode. Default value is '${MODE_APPEND}'"
+  echo "         * ${MODE_APPEND} - appends existed pages' comment by text specified using ${ARG_COMMENT}. '$CRADLE_ADMIN_DEFAULT_COMMENT' default page comment is removed"
+  echo "           Final comment has JSON string array format, for example: '[\"<existed comment>\",\"<specified comment>\"]'"
+  echo "         * ${MODE_SET} - sets text specified using ${ARG_COMMENT} as pages' comment"
+  echo "           Final comment has JSON string array format, for example: '[\"<specified comment>\"]'"
+  echo "         * ${MODE_RESET} - resets pages' comment to '$CRADLE_ADMIN_DEFAULT_COMMENT' default page comment"
+  echo "           Final comment is '$CRADLE_ADMIN_DEFAULT_COMMENT'"
+}
+
+parse_args() {
+  export CRADLE_ADMIN_TOOL_URL=''
+  export BOOK=''
+  export START_TIMESTAMP=''
+  export END_TIMESTAMP=''
+  export COMMENT=''
+  export MODE="${MODE_APPEND}"
+
+  while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+      --help)
+        args_help
+        exit 0
+        ;;
+      "${ARG_CRADLE_ADMIN_TOOL_URL}")
+        CRADLE_ADMIN_TOOL_URL="$2"
+        export CRADLE_ADMIN_TOOL_URL
+        shift 2
+        ;;
+      "${ARG_BOOK}")
+        BOOK="$2"
+        export BOOK
+        shift 2
+        ;;
+      "${ARG_START_TIMESTAMP}")
+        START_TIMESTAMP="$2"
+        export START_TIMESTAMP
+        shift 2
+        ;;
+      "${ARG_END_TIMESTAMP}")
+        END_TIMESTAMP="$2"
+        export END_TIMESTAMP
+        shift 2
+        ;;
+      "${ARG_COMMENT}")
+        COMMENT="$2"
+        export COMMENT
+        shift 2
+        ;;
+      "${ARG_MODE}")
+        MODE="$2"
+        export MODE
+        shift 2
+        ;;
+    esac
+  done
+
+  echo 'Arguments:'
+  echo " Arg: ${ARG_CRADLE_ADMIN_TOOL_URL} = '${CRADLE_ADMIN_TOOL_URL}'"
+  echo " Arg: ${ARG_BOOK} = '${BOOK}'"
+  echo " Arg: ${ARG_START_TIMESTAMP} = '${START_TIMESTAMP}'"
+  echo " Arg: ${ARG_END_TIMESTAMP} = '${END_TIMESTAMP}'"
+  echo " Arg: ${ARG_COMMENT} = '${COMMENT}'"
+  echo " Arg: ${ARG_MODE} = '${MODE}'"
+
+  verify_args
+}
+
+verify_args() {
+  echo 'Check arguments:'
+  verify_url
+  verify_book
+  verify_timestamps
+  verify_comment
+  verify_mode
+}
+
+verify_url() {
+  local url
+  url="${CRADLE_ADMIN_TOOL_URL}/${CRADLE_ADMIN_GET_ALL_BOOKS_PATH}"
+  echo " INFO: GET - $url"
+  if curl --head --silent "${url}" | head --lines 1 | grep --silent "200 OK"; then
+    echo " INFO: '${CRADLE_ADMIN_TOOL_URL}' URL is accessible"
+  else
+    echo " ERROR: '${CRADLE_ADMIN_TOOL_URL}' URL is not accessible"
+    args_help
+    exit 2
+  fi
+}
+
+verify_book() {
+  if [ -z "${BOOK}" ]; then
+    echo " ERROR: '${BOOK}' book can't be empty"
+    exit 3
+  fi
+
+  local book_list
+  url="${CRADLE_ADMIN_TOOL_URL}/${CRADLE_ADMIN_GET_ALL_BOOKS_PATH}"
+  echo " INFO: GET - $url"
+  book_list=$(curl --silent "${url}" | jq -r ".[].$CRADLE_ADMIN_BOOK_ID_KEY" | paste --serial --delimiters ' ')
```

Comment:
suggestion: I think we need to check for `jq` availability separately and write a message that it is required if it is not available (it is not a standard utility and it might not be installed on the host)